### ISSUE TITLE
Rodada-1.csv 2022 sem valores de scouts

### ISSUE THREAD

**Issue by rrasantos** [henriquepgomide/caRtola#88]
Boa noite @henriquepgomide, pelo que pude perceber o arquivo referente a rodada 1 do ano de 2022 não possui os valores de scouts, ele está igual a rodada-0. É isso mesmo? Caso precise, tenho salvo, em Excel, a rodada 1 extraída da api do cartola.

**Comment by arnaldog12** [comment]
Por favor, @rrasantos 
Iria ajudar muito!

**Comment by macielfjv** [comment]
@rrasantos , poderia compartilhar os dados da rodada-1?

**Comment by rrasantos** [comment]
[Rodada-1.csv](https://github.com/henriquepgomide/caRtola/files/8646094/Rodada-1.csv)

@macielfjv e @henriquepgomide desculpem pela demora. Vejam se é possível acessar os dados da rodada 1 no link acima.

**Comment by rrasantos** [comment]
No arquivo que postei anteriormente, encontrei erros ao salvar em CSV (nomes com acentos foram alterados).

@arnaldog12 e @macielfjv estou postando novamente o arquivo.

[Rodada-1.xlsx](https://github.com/henriquepgomide/caRtola/files/8646099/Rodada-1.xlsx)

**Comment by lima1986** [comment]
as p

> No arquivo que postei anteriormente, encontrei erros ao salvar em CSV (nomes com acentos foram alterados).
> 
> @arnaldog12 e @macielfjv estou postando novamente o arquivo.
> 
> [Rodada-1.xlsx](https://github.com/henriquepgomide/caRtola/files/8646099/Rodada-1.xlsx)

as pontuações nao estão separadas com virgulas, os números estão todos juntos.

**Comment by rrasantos** [comment]
> as p
> 
> > No arquivo que postei anteriormente, encontrei erros ao salvar em CSV (nomes com acentos foram alterados).
> > @arnaldog12 e @macielfjv estou postando novamente o arquivo.
> > [Rodada-1.xlsx](https://github.com/henriquepgomide/caRtola/files/8646099/Rodada-1.xlsx)
> 
> as pontuações nao estão separadas com virgulas, os números estão todos juntos.

Sim, não editei essa parte de pontuação, ela está nativa da api oficial, uma vez que em minha rotinas utilizo apenas os scouts gerados na rodada.

**Comment by macielfjv** [comment]
@rrasantos , obrigado pela planilha!! Consegue os arquivos das outras rodadas?

Alguns dos arquivos estão com problemas na pontuação, exemplo na sexta rodada as pontuações estão desagregadas e está gerando uma base errada quando rodo o script create_players_stats.R
